bitnami/thanos/values-production.yaml

```diff
@@ -0,0 +1,2037 @@
+## Global Docker image parameters
```

Comment:
This should not be included, please rebase master.

Reply:
https://github.com/bitnami/charts-docs/blob/main/DESIGN_DECISIONS.md#2021-01-20-removal-of-values-productionyaml-files